src/server/handlers.rs

```diff
@@ -762,3 +747,158 @@ impl<P: Publisher> Handler<P> {
         self.send_reply(reply, response).await;
     }
 }
+
+pub async fn open_schema_file() -> anyhow::Result<BufReader<File>> {
+    let schema_file = File::open("./oam/oam.schema.json")?;
+    Ok(BufReader::new(schema_file))
+}
+
+// Manifest validation
+pub(crate) fn validate_manifest(
+    manifest: Manifest,
+    current_manifests: StoredManifest,
+) -> Option<String> {
+    // let reader = open_schema_file().await.unwrap();
+    // let json_schema = serde_json::from_reader(reader).unwrap();
+    // let json_instance = serde_json::to_value(manifest.clone()).unwrap();
+    // let compiled_schema = JSONSchema::options()
```

Comment:
Looks like we may not have a JSON schema yet, depending on your answer above. If we don't, I would move this to a separate issue and remove the commented code